docs/release_notes.md

```diff
@@ -34,6 +36,11 @@
 - Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialisation. *AdonisFX-1165*
 - Fixed paint flood operation not normalising the weights properly. *AdonisFX-1146*
 
+### Known Limitations
+
+- The *Max Sliding Distance* parameter in AdnSkin and AdnSimshape is represented in scene units. The higher this value is, the more units in the space the sliding constraint allows a vertex to slide on. If the polygons of the target geometry to slide on are very little compared to that value, then the sliding constraint will be time and memory consuming. *AdonisFX-997*
```

Comment:
There is no issue yet but should we mention that we do not support simulated muscles as geometry targets? @carlosmonteagudoinbibo 

We already have a note at Targets from muscle.md and ribbon.md

Reply:
I've already added as an improvement.